pymisp/mispevent.py

```diff
@@ -656,10 +657,7 @@ def __init__(self, name: str, strict: bool=False, standalone: bool=False, defaul
         else:
             self.distribution = 5  # Default to inherit
             self.sharing_group_id = 0
-        self._standalone = standalone
-        if self._standalone:
-            # Mark as non_jsonable because we need to add the references manually after the object(s) have been created
-            self.update_not_jsonable('ObjectReference')
+        self.standalone = standalone
```

Comment:
is this a typo?

Reply:
Setting `self.standalone` to `True` also adds "ObjectReference" to `__not_jsonable`, the initialization earlier on is needed because `standalone.setter` first checks `_standalone`